Sources/BuildArtifacts/BuildArtifacts.swift

```diff
@@ -12,24 +12,30 @@ public struct BuildArtifacts: Codable, Hashable, CustomStringConvertible {
     
     /// Location of additional apps that can be launched diring tests.
     public let additionalApplicationBundles: [AdditionalAppBundleLocation]
+    
+    /// Dictionary with Host : Token for artifacts urls
+    public let hostsTokens: [String: String]
```

Comment:
Let's extend this feature a bit. Can you please rename this to `hostHeaders: [String: [String: String]]` to allow appending an arbitrary HTTP headers to outgoing requests.